pyspecdata/core.py

```diff
@@ -578,9 +578,10 @@ def lookup_rec(A, B, indexpair):
             )
         )
     whichisindex = joined[0][1].dtype.names.index(indexpair[1])
-    allbutindex = (
-        lambda x: list(x)[0:whichisindex] + list(x)[whichisindex + 1 :]
-    )
+
+    def allbutindex(x):
+        return list(x)[0:whichisindex] + list(x)[whichisindex + 1 :]
+
```

Comment:
it seemed that there was nothing in core.py that we would want, so I just reverted to the version that's currently in core (after flake8 compliance)